daisy_workflows/image_test/metadata-script/metadata-script-test-shutdown-hash.sh

```diff
@@ -201,3 +199,10 @@ md5sum "${0}" | logger -p daemon.info
 # PhjxoxKKfSvpfTOH1Ofi450SyGIz497dm0CncRcFP+cARc+FatNT48ABy1RLHpJMJg0ae5+VKEli
 # 6txosGOmsciVWXjn1m3tlZLrKtRW4b8E7qzCsvuHGkcp8PSo4jDlWrLueNRvzWmTr7CzvDXroFHY
 # gZFvZIyZIO1TA4mXJY8nEh10yU5MmoqMv4Gu7KbSS6dtGdd/qg==
+EOF
+
+chmod +x shutdown_tester.sh
+#./shutdown_tester.sh
```

Comment:
Is this supposed to be commented out?

Reply:
no. I left test code there and didn't notice. Just a sec

Reply:
done